### ISSUE TITLE
Make timelimit command more intuitive to use

### ISSUE THREAD

**Issue by Aciz** [etlegacy/etlegacy#1689]
`timelimit` commands fraction part should be absolute seconds rather than a percentage of 60 seconds, to make easier to set clock to a precise time, if for example a stopwatch game has been reset after one round.

**Comment by rmarquis** [comment]
Suggestion: keep current behaviour when decimal value like `3.5` are passed, but transform to second when `3:30` are used.

**Comment by Aciz** [comment]
Yes, it's probably a good idea to keep the old behavior for those who have gotten accustomed to it over the years, but also provide a simpler, more user friendly solution.
